.github/workflows/manifest.yml

```diff
@@ -53,3 +53,5 @@ jobs:
           verbosity-level: '1'
           labels: 'manifest'
           dnm-labels: 'DNM (manifest)'
+          blobs-added-labels: 'Binary Blobs Added'
+          blobs-modified-labels: 'Binary Blobs Modified'
```

Comment:
should one of these be a DNM one?

Reply:
The thing is that the action already sets "DNM (manifest)" whenever `module.yml` is modified anyway, so I don't think it's required.